SuperBuild.cmake

```
@@ -0,0 +1,16 @@
+include (ExternalProject)
+
+
+set (DEPENDENCIES)
+list (APPEND DEPENDENCIES TIFF)
+include(${PROJECT_SOURCE_DIR}/Superbuild/External_LIBTIFF.cmake)
+list (APPEND DEPENDENCIES GLEW)
+include(${PROJECT_SOURCE_DIR}/Superbuild/External_GLEW.cmake)
```

```suggestion
set(DEPENDENCIES
  GLEW
  TIFF
)
foreach(dependency IN LISTS DEPENDENCIES)
  message("SuperBuild - Adding ${dependency}")
  include(${PROJECT_SOURCE_DIR}/SuperBuild/External_${dependency}.cmake)
endif()
```

* [ ] Rename directory from `Superbuild` to `SuperBuild`